shell/edgexfoundry-snapcraft.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash -x
+
+# note that since this script gets included into a yaml string using jjb's
+# include-raw-escape macro, we can't use any squiggly braces here, otherwise
+# they will get duplicated
+# for some reason we also can't use include-raw macro because that macro will
+# attempt to interpret anything contianed in a macro as a jenkins key to
+# expand which we also don't want
+
+# if we are running inside a jenkins instance then copy the login file 
+# and check if this is a release job
+if [ -n "$JENKINS_URL" ]; then
+    # since we are running from jenkins, use the workspace as the git root
+    GIT_ROOT=$WORKSPACE
+    if [ -f "$HOME/EdgeX" ]; then
+       cp "$HOME/EdgeX" "$GIT_ROOT/edgex-snap-store-login"
+    else
+        echo "I seem to be running on Jenkins, but there's not a snap store login file..." 
```

Comment:
Just want to confirm that we don't want to abort here, but just echo this warning and continue on.

Reply:
The reason not to exit is mainly just for convenience in testing on the sandbox and locally so that I can test locally by running (from the `edgex-go` dir cloned locally):
```bash
JENKINS_URL=something JOB_NAME=edgex-go-snap-master-stage-snap /home/me/git/ci-management/shell/edgexfoundry-snapcraft.sh
```
and have the job continue to run.

We could certainly make it abort here too but then to test on the sandbox and locally I would need to either create a fake file at `$HOME/EdgeX` or modify the script